packages/server/src/services/feed-fires/feed-fires.class.ts

```diff
@@ -1,12 +1,102 @@
+import { BadRequest } from '@feathersjs/errors';
+import { Params } from '@feathersjs/feathers';
 import { Service, SequelizeServiceOptions } from 'feathers-sequelize';
+import { QueryTypes } from 'sequelize';
 import { Application } from '../../declarations';
+import { extractLoggedInUserFromParams } from '../auth-management/auth-management.utils';
 
 /**
  * A class for ARC Feed service
  */
 export class FeedFires extends Service {
   //eslint-disable-next-line @typescript-eslint/no-unused-vars
+  app: Application
+  docs: any
+
   constructor(options: Partial<SequelizeServiceOptions>, app: Application) {
     super(options);
+    this.app = app;
+  }
+
+    /**
+   * @function find it is used to find specific users
+   *
+   * @param params user id
+   * @returns {@Array} of found users
+   */
+
+     async find (params: Params): Promise<any> {
+      const loggedInUser = extractLoggedInUserFromParams(params);
+      if (!loggedInUser.userId) {
+        return Promise.reject(new BadRequest('Could not add fire. Users isn\'t logged in! '));
```

Comment:
i don't think we need this rejection here, or at least change it to "Could not get fires"